modules/envvars.nix

```diff
@@ -0,0 +1,25 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = (
```

Comment:
I think there should be
```
mkIf (home.sessionVariableSetter == "shell" || home.sessionVariableSetter == "bash")
```
here. If the user said to use PAM then I think it is proper not to create these files.